packages/forms/src/UIForm/fields/Date/Time.component.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { InputTimePicker } from '@talend/react-components/lib/DateTimePickers';
+
+import FieldTemplate from '../FieldTemplate';
+import { generateDescriptionId, generateErrorId } from '../../Message/generateId';
+
+function TimeWidget(props) {
```

Comment:
you can destruct props here instead
```suggestion
function TimeWidget({
		errorMessage,
		id,
		isValid,
		options = {},
		onChange,
		onFinish,
		schema,
		value,
		valueIsUpdating,
	}) {
```